soh/soh/Enhancements/randomizer/option_descriptions.cpp

```diff
@@ -0,0 +1,523 @@
+#include "settings.h"
```

Comment:
Bit of a tangent, obviously a much larger conversation, but should we consider moving this type of static string constant data to soh.otr or something? Might be easier to facilitate translations that way.